app/assets/javascripts/host_checkbox.js.erb

```diff
@@ -40,13 +40,17 @@ function readFromCookie() {
 }
 
 function toggle_actions() {
-  var dropdown = $("#submit_multiple a");
+  var dropDownContainer = $("#submit_multiple");
+  var dropdown = dropDownContainer.find("a");
+  var disabledMessage = __("Please select hosts to perform action on.");
```

Comment:
we will still need to keep the code at host_checkbox.js, since it handles removing and adding the title when hosts are selected/unselected (we dont want the title for an active dropdown)